### PR TITLE
Allow db:schema:load or db:structure:load

### PR DESCRIPTION
**Pull request by rdunlop**

### CODE REVIEW

File: src/standard.yml [hunk 1]
```diff
@@ -131,7 +132,8 @@ jobs:
 
       # Setup DB in this job, because it cannot be shared through the attach_workspace
       - run: bundle exec rake db:create
-      - run: bundle exec rake db:schema:load
+      - run: if [ -e db/schema.rb ]; then bundle exec rake db:schema:load; fi
+      - run: if [ -e db/structure.sql ]; then bundle exec rake db:structure:load; fi
```
Comment (rdunlop) [inline src/standard.yml:147]: I need to fix this so that it loads the `mysql` or `postgres` library before running.

Comment (giladshanan) [reply src/standard.yml:147]: @rdunlop let me know when you've added the fix and I'll take another look

Comment (rdunlop) [reply src/standard.yml:147]: @giladshanan ready for re-review

**Review by giladshanan — APPROVED** [review]
👍

**Pull request merged**